conf/pretalx.cfg

```diff
@@ -2,7 +2,9 @@
 # use PRETALX_FILESYSTEM_* env vars
 
 [site]
-; never run debug in production
+; never run debug in production. Make sure you serve all requests for the
+; `/static/` and `/media/` paths when debug is False.
+; See [installation](https://docs.pretalx.org/administrator/installation/#step-7-ssl) for more information
```

Comment:
Not sure it's particularly useful to have a markdown-formatted link in a file that will never be parsed as markdown, and will make the link harder to select and open in most file editors

Reply:
Yes, you're absolutely right about the markdown... Should i remove it?